nemo_text_processing/inverse_text_normalization/inverse_normalize.py

```diff
@@ -89,7 +94,7 @@ def inverse_normalize(self, text: str, verbose: bool) -> str:
 def parse_args():
     parser = ArgumentParser()
     parser.add_argument("input_string", help="input string", type=str)
-    parser.add_argument("--language", help="language", choices=['en', 'de', 'es', 'ru'], default="en", type=str)
+    parser.add_argument("--language", help="language", choices=['en', 'de', 'es', 'ru', 'fr'], default="en", type=str)
```

Comment:
Could you all add `fr` option to tools/deployment.. scripts?

Reply:
Should be there already, no? I thought i updated pynini_export earlier.

Reply:
https://github.com/NVIDIA/NeMo/blob/main/tools/text_processing_deployment/export_grammars.sh#L35

Reply:
Ah!